frontend/src/components/pages/CampCreation/index.tsx

```diff
@@ -19,20 +20,30 @@ const CampCreationPage = (): React.ReactElement => {
   const [campDetailsDummyTwo, setCampDetailsDummyTwo] = useState(false);
   const [campDetailsDummyThree, setCampDetailsDummyThree] = useState("");
 
-  const [scheduledSessions, setScheduledSessions] = React.useState<
-    CreateCampSession[]
-  >([]);
+  const [scheduledSessions, setScheduledSessions] = React.useState<CreateCampSession[]>([]);
 
   const [visitedRegistrationPage, setVisitedRegistrationPage] = useState(false);
 
   // Variables to determine whether or not all required fields have been filled out.
   // NOTE: This will depend on what type of state a page requires, i.e. determining
   // if a checkbox is checked is different than determining if an input field is filled.
-  const isCampDetailsFilled =
-    campDetailsDummyOne && campDetailsDummyTwo && campDetailsDummyThree !== "";
+  const isCampDetailsFilled = campDetailsDummyOne && campDetailsDummyTwo && campDetailsDummyThree !== "";
```

Comment:
Did linter not want two lines?

Reply:
I put that there before (just for my own sake) but now I've re-enabled the linter on those lines so it shouldn't be a problem anymore.